eng/scripts/mark-shipped.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell -noprofile -executionPolicy Bypass -file "%~dp0\mark-shipped.ps1"
```

Comment:
Just curious: @tmat how did the https://github.com/dotnet/roslyn/tree/main/scripts/PublicApi files work with `-executionPolicy RemoteSigned`❔